examples/material_gallery/lib/gallery/item.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+typedef Widget GalleryDemoBuilder();
+
+class GalleryItem extends StatelessWidget {
+  GalleryItem({ this.title, this.icon, this.builder });
+
+  final String title;
+  final IconData icon;
+  final GalleryDemoBuilder builder;
+
+  Widget build(BuildContext context) {
+    Widget leading = null;
```

Comment:
The value of leading (below) can just be `icon != null ? new(Icon: icon) : null`